CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Fixed
+
+- Fixed an issue deploying content to free Posit Connect Cloud accounts with incorrect  
+  access settings.
```

Comment:
I'm a bit late to the merge, but did we want to cite the issue here?

Reply:
Yeah. Fixed that in https://github.com/posit-dev/publisher/pull/3010.